spec/classroom_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../classroom'
+
+describe Classroom do
+  before(:each) do
+    @book = Classroom.new('Classroom')
+  end
```

Comment:
- [ ] The naming of @book variable is quite confusing because you're basically saying that `@book` is an instance of the Classroom class. I think you meant to say `@classroom = Classroom.new('Classroom')` ? 
- [ ] Please test the `add_student` method in your `classroom.rb` file 🙏